Allan690/Store-Manager
# #161783456 Implement logout functionality

#### What does this PR do?
Implements logout functionality for the application
#### Description of Task to be completed?
Invalidating the token used for access.
#### How should this be manually tested?
1. Clone the repo
2. Checkout to the ft-logout-functionality branch
3. Login to the application using the default admin credentials:
`allan@gmail.com` and `allangmailcompany`
4. Log out by clicking the logout button on the navbar
5. Now try to create a user. The application should raise an alert requiring you to log in as an admin.

#### Any background context you want to provide?
None
#### What are the relevant pivotal tracker stories?
#161783456
#### Questions: Feedback on how I can improve this?
